### DISCUSSION TITLE
FindNeighbors function get error

### DISCUSSION THREAD

**Discussion by LitalBarak96** [satijalab/seurat#6762] [Q&A]
whenever I am trying to use the function this is what happening
Computing nearest neighbor graph
Computing SNN
Error in validityMethod(as(object, superClass)) : 
  object 'CsparseMatrix_validate' not found

i tried to remove and install again the Matrix and Seurat package but it didn't work.

please help

**Comment by Ramirescu** [comment]
Same error trying the first tutorial. With R 4.2.2, RStudio and Seurat 4.0.6. Tried reintalling R and Rstudio and the error still apears.

**Reply by Ramirescu** [reply]
> whenever I am trying to use the function this is what happening Computing nearest neighbor graph Computing SNN Error in validityMethod(as(object, superClass)) : object 'CsparseMatrix_validate' not found
> 
> i tried to remove and install again the Matrix and Seurat package but it didn't work.
> 
> please help

**I Solved it by intalling R 4.1.3 Instead of 4.2.2.**
There are still lots of issues with the pbmc3k tutorial downstream of this one, this is just the start...

**Reply by Ramirescu** [reply]
Also solved by @samuel-marsh ,
I removed the package and reinstalled the latest version of Matrix by downloading the binary file from CRAN.
It has now upgraded to Matrix version 1.5-4 and the problem is sorted.
Thanks a lot.
